.github/workflows/windows-packages.yaml

```diff
@@ -10,14 +10,51 @@ on:
     branches:
     - release_test
 jobs:
+  config:
+    runs-on: ubuntu-latest
+    outputs:
+      pg12_earliest: ${{ steps.config.outputs.pg12_abi_min }}
+      pg13_earliest: ${{ steps.config.outputs.pg13_abi_min }}
+      pg14_earliest: ${{ steps.config.outputs.pg14_abi_min }}
```

Comment:
I don't get it, where do we get `pg??_abi_min` from?

Reply:
Was this supposed to be `pg??_earliest` instead?

Reply:
https://github.com/timescale/timescaledb/pull/4638/files